cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt

```diff
@@ -10,14 +10,12 @@ errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagn
 errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
 vim ex 'e main.go'
-errlogmatch -peek 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
 vim ex 'e other.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
-errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
-errlogmatch -count=2 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
+errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
```

Comment:
Is this really how things should work? (I notice that the tests passed so I assume it matches the gopls behaviour here). Aren't we supposed to get diagnostics when we open a file, even if it's just empty?

If the code is indeed correct, we need to update the test description at the top of this file.

Reply:
> Aren't we supposed to get diagnostics when we open a file, even if it's just empty?

Not any more. So yes, this is now correct.

Reply:
Updated the description in this and other files. Thanks for that catch.